<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>практика 1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>информация о системе</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>работа с фалами</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>открыть</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>сохранить</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>удалить файл</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>json</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>список студентов</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>ФИО</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Год поступления</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>направление</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="QListWidget" name="listWidget_2"/>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>прочитать</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>сохранить json</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>добавить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>xml</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>список дисциплин</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>год утверждения</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="3">
         <widget class="QLineEdit" name="lineEdit_5"/>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
        <item row="7" column="0" colspan="3">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>добавить</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="3">
         <widget class="QListWidget" name="listWidget_3"/>
        </item>
        <item row="6" column="0" colspan="3">
         <widget class="QLineEdit" name="lineEdit_6"/>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>название</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="3">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>преподаватель</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="3">
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>сохранить</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="3">
         <widget class="QPushButton" name="pushButton_8">
          <property name="text">
           <string>открыть</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>архивы</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_11">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>разархивировать</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_13">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>создать архив</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QListWidget" name="listWidget_4"/>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_12">
          <property name="text">
           <string>добавить файл в архив</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_10">
          <property name="text">
           <string>открыть архив</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>брутфорс</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>многопоточный подбор</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>1115dd800feaacefdf481f1f9070374a2a81e27880f187396db67958b207cbad</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>3a7bd3e2360a3d29eea436fcfb7e44c735d117c42d1c1835420b6b9942dd4f1b</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>74e1bb62f8dabb8125a58852b63bdf6eaef667cb56ac7f7cdba6d7305c50a22f</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_15">
            <property name="text">
             <string>вычислить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>результат</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>однопоточный подбор</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>1115dd800feaacefdf481f1f9070374a2a81e27880f187396db67958b207cbad</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>3a7bd3e2360a3d29eea436fcfb7e44c735d117c42d1c1835420b6b9942dd4f1b</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>74e1bb62f8dabb8125a58852b63bdf6eaef667cb56ac7f7cdba6d7305c50a22f</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_14">
            <property name="text">
             <string>вычислить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>результат</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
